hack/validate/swagger

```diff
@@ -7,7 +7,12 @@ IFS=$'\n'
 files=( $(validate_diff --diff-filter=ACMR --name-only -- 'api/swagger.yaml' || true) )
 unset IFS
 
-if [ ${#files[@]} -gt 0 ]; then
-	LANG=C.UTF-8 yamllint -c ${SCRIPTDIR}/.swagger-yamllint api/swagger.yaml
-	swagger validate api/swagger.yaml
+if [ -n "${TEST_FORCE_VALIDATE}" ] || [ ${#files[@]} -gt 0 ]; then
```

Comment:
Same comment as above (should this be `${TEST_FORCE_VALIDATE:-}` in case this is run locally?)